modules/controller/command_parser.py

```diff
@@ -0,0 +1,38 @@
+from typing import List, Dict, Tuple
+
+from modules.controller.commands.classify_command import ClassifyCommand
+from modules.controller.commands.command import Command
+
+
+##  Class for parsing strings to a command
+from modules.controller.commands.collector_command import CollectorCommand
+from modules.controller.commands.label_command import LabelCommand
+from modules.controller.commands.quit_command import QuitCommand
+from modules.controller.commands.train_command import TrainCommand
+from modules.exception.excpetions import IllegalArgumentException
+
+
+class CommandParser:
+
+    _valid_commands: Dict[str, Command.__class__] = {
+        "collect": CollectorCommand,
```

Comment:
why is the naming for the command at the one hand a substantive (labeler) but on the other hand a verb (collect)?
Isn't that inconsistent?

Reply:
I changed it in the new commit, please review